src/NUnitFramework/framework/Constraints/ConstraintExpression.cs

```diff
@@ -45,7 +45,7 @@ public class ConstraintExpression
         /// <summary>
         /// The ConstraintBuilder holding the elements recognized so far
         /// </summary>
-        protected ConstraintBuilder builder;
+        protected readonly ConstraintBuilder builder;
```

Comment:
Noticed that I missed this in the `readonly` PR. Maybe I should put this in its own PR?